.github/CODEOWNERS

```diff
@@ -1,2 +0,0 @@
-# Guessed from System and Package relationships in Biz Ops
-* @Financial-Times/apps
```

Comment:
Interesting! I believe this should definitely have a codeowners file associating it with our team. 

E.g. https://github.com/financial-times/ft-app-model

What was the error? (This example looks valid. https://github.com/Financial-Times/ft-app-model/commit/a8d27840b5d22d0e028872d611cc57e2c88a61f8)

Reply:
I think it's something about inherited permissions. For example, the ft-app file at https://github.com/Financial-Times/ft-app/blob/main/.github/CODEOWNERS has that error - because Customer Products has Admin, and Apps is part of Customer Products 🙄

Reply:
Rather than reverting all these changes, it would be better to close this PR.

Will do after standup - shout out if you all think there is some value to be added here.